gnovm/stdlibs/testing/testing.gno

```diff
@@ -326,7 +328,9 @@ func tRunner(t *T, fn testingFunc, verbose bool) {
 	if !t.shouldRun(t.name) {
 		return
 	}
-
+	// m := gno.NewMachine("", nil)
```

Comment:
Are these lines meant to be commented out?

Reply:
Thanks for this detail review. I removed all the comment that I used for testing purpose.